### DISCUSSION TITLE
Will it work for GitLab?

### DISCUSSION THREAD

**Discussion by jackez** [ejoffe/spr#265] [Q&A] [answered]
This tool looks very promising! Thanks for working on it! Will it work for GitLab too? If not, is there any plans to add GitLab support?

**Comment by leoluk** [comment, marked as answer]
Won't work in Gitlab.

In case you want to self-host, consider Gerrit :) It works like spr, but nicer.

**Reply by ejoffe** [reply]
spr uses the GitHub api to create and manage pull requests. 
I suppose the interface could be implemented for GitLab if there is a similar api that could be used. 
There is currently no plan to add this support, someone using gitlab would have to implement this feature.
